guava/src/com/google/common/hash/BloomFilter.java

```diff
@@ -337,7 +343,7 @@ public int hashCode() {
    * @since 23.0
    */
   public static <T> Collector<T, ?, BloomFilter<T>> toBloomFilter(
-      Funnel<? super T> funnel, long expectedInsertions, double fpp) {
+      Funnel<? super T> funnel, @Positive long expectedInsertions, @IntRange(from = 0, to = 1) double fpp) {
```

Comment:
This annotation is wrong. `@IntRange` cannot be applied to doubles, and there is no double range annotation in the Value Checker.

Reply:
There should be no need to annotate doubles for the Index Checker, anyway.